content/docs/guides/vercel.md

```diff
@@ -131,34 +130,51 @@ If you added the Neon Vercel Integration to a single Vercel project but would li
 Please be aware that the Neon Vercel Integration connects one Vercel project to one Neon project. It does not support connecting multiple Vercel projects to one Neon project, nor does it support connecting multiple Neon projects to one Vercel project. The steps below outline how to make the integration available to other Vercel projects to use with their own separate and dedicated Neon project.
 </Admonition>
 
-1. Ensure that the Neon Vercel Integration that you added previously has access to the Vercel project.
+1. Ensure that the Neon Vercel Integration that you added previously has access to the Vercel project that you want to use with the Neon Vercel Integration.
```

Comment:
Consider changing "Ensure" to a less formal "Make sure"

Reply:
Changed. Thank you.